### ISSUE TITLE
Question about padding in PatchEmbed

### ISSUE THREAD

**Issue by Pukei-Pukei** [ViTAE-Transformer/ViTPose#70]
self.proj = nn.Conv2d(in_chans, embed_dim, kernel_size=patch_size, stride=(patch_size[0] // ratio), padding=4 + 2 * (ratio//2-1))
[ViTPose/mmpose/models/backbones/vit.py#L157](https://github.com/ViTAE-Transformer/ViTPose/blob/4fd8507ad3b1d1c7e0cd1491cf3f4ca82c35204f/mmpose/models/backbones/vit.py#L157)

In the base setting, img_size=(256, 192) is already divisible by patch_size=16.
So I think padding should be 0 instead of 2=4+2*(ratio//2-1) where ratio=1.
I hope you will let me know if I missed anything.

**Comment by Annbless** [comment]
Thanks for your pointing. Removing the padding in your experiments for training the models is free and harmless.

**Comment by Annbless** [comment]
It seems there are no further questions. I will close this issue temporarily. If you have some further questions, please feel free to re-open it.
